ledger/participant-state/kvutils/src/test/suite/scala/com/daml/ledger/participant/state/kvutils/committer/transaction/TransactionCommitterSpec.scala

```diff
@@ -866,43 +267,16 @@ class TransactionCommitterSpec extends AnyWordSpec with Matchers with MockitoSug
       inStaticTimeMode = false,
     )
 
-  private def contextWithTimeModelAndEmptyCommandDeduplication() =
-    createCommitContext(recordTime = None, inputs = inputWithTimeModelAndEmptyCommandDeduplication)
-
-  private def contextWithTimeModelAndCommandDeduplication() =
-    createCommitContext(recordTime = None, inputs = inputWithTimeModelAndCommandDeduplication)
-
   private def newDedupValue(deduplicationTime: Timestamp): DamlStateValue =
     DamlStateValue.newBuilder
       .setCommandDedup(
         DamlCommandDedupValue.newBuilder.setDeduplicatedUntil(buildTimestamp(deduplicationTime))
       )
       .build
 
-  private def createProtobufTimestamp(seconds: Long) =
-    Conversions.buildTimestamp(Timestamp.assertFromInstant(Instant.ofEpochSecond(seconds)))
-
-  private def defaultConfigurationStateValueBuilder(): DamlStateValue.Builder =
-    DamlStateValue.newBuilder
-      .setConfigurationEntry(
-        DamlConfigurationEntry.newBuilder
-          .setConfiguration(Configuration.encode(theDefaultConfig))
-      )
-
   private def createEmptyTransactionEntry(submitters: List[String]): DamlTransactionEntry =
     createTransactionEntry(submitters, TransactionBuilder.EmptySubmitted)
 
-  private def createTransactionEntry(submitters: List[String], tx: SubmittedTransaction) =
-    DamlTransactionEntry.newBuilder
-      .setTransaction(Conversions.encodeTransaction(tx))
-      .setSubmitterInfo(
-        DamlSubmitterInfo.newBuilder
-          .setCommandId("commandId")
-          .addAllSubmitters(submitters.asJava)
-      )
-      .setSubmissionSeed(ByteString.copyFromUtf8("a" * 32))
-      .build
-
   private def tuple(values: String*): TransactionBuilder.Value =
```

Comment:
You might want to move this into `TestHelpers` as well.

Reply:
Done.